### DISCUSSION TITLE
Implement a "Cost of Transmission" calculation

### DISCUSSION THREAD

**Discussion by malcolm-dsider** [NatLabRockies/GEOPHIRES-X#114] [Ideas]
Shall we implement a "cost of transmission" calculation modeled on the "cost of field gathering system" calculation? The total cost of transmission infrastructure is often required to be included in the total cost of a project. 
Default behavior would be for no cost of transmission by default, but it would be included in all financial calculations (NPV, IRR, etc.) and reported when it was non-zero. So no change to any default behaviors and examples, except when explicitly called for.
Inputs would be:
Transmission System Capital Cost
o Description: Total transmission system capital cost per km (including interconnection and substation costs)
o Parameter type: Real
o Units: M$
o Allowable value range: [0,100]
o When required: Optional
o Default value: 1
Transmission System Length
o Description: Length of transmission system that must be built
o Parameter type: Real
o Units: N/A
o Allowable value range: [0,10]
o Default value: 0 (i.e., no transmission would be built)
Transmission System Capital Cost Adjustment Factor
o Description: Multiplier for built-in transmission system capital cost correlation
o Parameter type: Real
o Units: N/A
o Allowable value range: [0,10]
o Default value: 1

**Comment by softwareengineerprogrammer** [comment]
This was implemented as of the port that marks the start of the current repository's history, see https://github.com/NREL/GEOPHIRES-X/blob/7d1419b07c84e1929d053efd27a4622a5c19c0ab/src/geophires_x/Economics.py#L1650-L1656
